app/models/mixins/ownership_mixin.rb

```diff
@@ -11,9 +11,27 @@ module OwnershipMixin
     virtual_column :owned_by_current_user,                :type => :boolean,    :uses => :evm_owner_userid
     virtual_column :owning_ldap_group,                    :type => :string,     :uses => :miq_group
     virtual_column :owned_by_current_ldap_group,          :type => :boolean,    :uses => :owning_ldap_group
+
+    # Determine whether to return vms ownded by the current user or not
```

Comment:
Yeah... need turn on my [spellchecker](https://github.com/NickLaMuro/dotfiles/blob/master/.vimrc.d/spellcheck.vim#L48-L49) on by default for ruby files too I guess 😞